packages/base/adapters/evm/wagmi/connectors/AuthConnector.ts

```diff
@@ -1,5 +1,6 @@
 import { createConnector, type CreateConfigParameters } from '@wagmi/core'
 import { W3mFrameProvider } from '@web3modal/wallet'
+import { ConstantsUtil as CommonConstantsUtil } from '@web3modal/common'
```

Comment:
We have various ConstantsUtil across our code base. Could we unify some of them or could we use a more specific naming so we have a bit more context when we use them.

Reply:
💯 I say let's note it and do it separately as not to block this merge